### DISCUSSION TITLE
Welcome to SLEAP Discussions!

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#728] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eringiglio** [comment]
Hi! I'm working with SLEAP on a very large, dense dataset of mouse recordings (n = ~1000 90min videos). I'd like to have a place to ask more questions about using SLEAP on remote machines, because I'm swiftly reaching the point that I'm outpacing the computing capacity of my local machine.
